README.md

```diff
@@ -12,8 +12,20 @@
 For installing this plugin, add the following line to your `plugins.sbt` file:
 
 ```scala
-addSbtPlugin("io.higherkindness" % "sbt-mu-srcgen" % "0.22.3")
+addSbtPlugin("io.higherkindness" % "sbt-mu-srcgen" % "$version") // see badge in the project for the latest version
```

Comment:
The readme is auto-generated on release, so you need to make your changes in `docs/README.md` instead

Reply:
oh yeah that makes sense!  I saw both readmes and I wasn't sure which one to edit; thanks for clarifying.